x-pack/plugins/index_management/public/application/components/component_templates/component_template_details/component_template_details.tsx

```diff
@@ -109,11 +117,27 @@ export const ComponentTemplateDetailsFlyout: React.FunctionComponent<Props> = ({
       maxWidth={500}
     >
       <EuiFlyoutHeader>
-        <EuiTitle size="m">
-          <h2 id="componentTemplateDetailsFlyoutTitle" data-test-subj="title">
-            {decodedComponentTemplateName}
-          </h2>
-        </EuiTitle>
+        <EuiFlexGroup alignItems="center" gutterSize="s">
+          <EuiFlexItem grow={false}>
+            <EuiTitle size="m">
+              <h2 id="componentTemplateDetailsFlyoutTitle" data-test-subj="title">
+                {decodedComponentTemplateName}
+              </h2>
+            </EuiTitle>
+          </EuiFlexItem>
+
+          {componentTemplateDetails?._kbnMeta.isManaged ? (
+            <EuiFlexItem grow={false}>
+              {' '}
```

Comment:
Is this whitespace intended to be included? Perhaps &npsb; would be better if so.

Reply:
Yes, it is intentional. I'm going to leave as is for now, but will consider changing in a follow-up PR.

Reply:
FYI the empty space that Alison used will allow a line break if the container forces the text to wrap, but a non-breaking space won’t allow this. In most situations I think we want the former behavior.

Reply:
Thanks for pointing this out @cjcenizal!

Reply:
> FYI the empty space that Alison used will allow a line break if the container forces the text to wrap, but a non-breaking space won’t allow this. In most situations I think we want the former behavior.

@cjcenizal Not sure I understand correctly. Are we talking about 1 space difference (probably a few pixels) that would not wrap? Ex: ` hello world`: where would this not wrap after "hello"?

Reply:
I’m not sure if this answers your question but I was just point out that `&nbsp;` !== `‘ ‘`. The former won’t allow a line break on the space (“hello world” will overflow its container) while the latter will (“hello world” will break into two lines consisting of “hello” and “world”).

Reply:
This is strange, I would need to try it out 😊 It would make sense if we were talking about `hello&nbsp;world` but here we are adding the &nbsp; before the 2 words.